src/spacel/provision/template/app.py

```diff
@@ -141,6 +143,22 @@ def app(self, app_region):
                 }
             }
 
+        # Custom update policy:
+        if app_region.update_policy == 'disabled':
+            del resources['Asg']['UpdatePolicy']
+        if app_region.update_policy == 'redblack':
+            resources['Asg']['UpdatePolicy'] = {
+                'AutoScalingReplacingUpdate': {
+                    'WillReplace': 'true'
+                }
+            }
+            time_pos = (resources['Lc']
```

Comment:
Do we still need this cache-buster?

Given that _everything_ in the `AppManifest` (units, environment variables) is already in UserData, I think redeploys should still trigger correctly without it.